BLSQ/iaso
# POLIO-1365: Track dashboard use: Move the analytics script field from page to account

Explain what problem this PR is resolving
- The analytics script should be one for each account
- For that, the field related to it should be moved from page to account
Related JIRA tickets : [POLIO-1365](https://bluesquare.atlassian.net/browse/POLIO-1365)

## Self proofreading checklist


## Changes
- Remove the analytics_script field from page model
- Add it to account model
- Add the field to api/setaccount

## How to test
- Setaccount or edit on from the admin
- Add an analytics script sample
- Go in the embedded links
- Check in inspect element if the script is in the head tag

## Print screen / video
![Screenshot from 2023-12-20 17-17-15](https://github.com/BLSQ/iaso/assets/19631540/705dd66a-3621-4676-85b5-593a8a1232b6)
![Screenshot from 2023-12-20 17-16-47](https://github.com/BLSQ/iaso/assets/19631540/2f131243-d4bc-4cec-9a19-a13ebe3adcb0)


[POLIO-1365]: https://bluesquare.atlassian.net/browse/POLIO-1365?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
